TESTING.md

```diff
@@ -24,58 +24,67 @@ Things to look at:
  
  *Blue:*
  - Selected/highlighted content
- - variable controls - progress bars, scales, level bars
+ - variable/ scaling controls - progress bars, scales, level bars
  - General Content-relevant controls
  
  *Orange:*
  - Binary on/off actions
  - radio buttons
- - active buttons
- - toggle switches
  - checkboxes
  - switches
  - Pathbars
+ - Focused entries
  - General context-display controls
   
  *Green:*
  - Save buttons once they can be clicked
  - Affirmative/progressive action buttons/controls
  
  *Header and title bars:* 
- - brown in normal theme
- - dark theme - darker brown
- - light theme - grey
+ - brown in normal theme (#574F4A)
+ - dark theme - dark grey (#363636)
+ - light theme - light grey (Light theme is currently missing)
  
  *Window Backgrounds:*
- - light grey/white background in normal theme
- - dark brown in dark theme
+ - light grey in normal theme (#F6F6F6)
+ - dark grey in dark theme (#303030)
  
- - progress spinner should invert colors in dark theme
- 
- **Check that Nautilus has the proper color**<a name="nautilus"></a>
- - context information (sidebar entries)
- - tabs have orange underlines, path bars
+ *Progress spinner/eyeball should invert colors in dark theme*
+
+ *Text/Input Colors*
+ - Very light grey in light theme, (#FCFCFC)
+ - Very dark grey in dark theme (#2A2A2A)
  
 **Gnome-Tweak-Tool**<a name="gtt"></a>
  
- *6 themes:*
-  - Pop{,-slim}
-  - Pop-Dark{,-slim}
-  - Pop-Light{,-slim}
+ *3 themes:*
+  - Pop
+  - Pop-Dark
+  - Pop-Light
  
  **Other things to note:**
- 
- - Everything has a shadow
  - Everything needs to be legible/high contrast
- - Bright colors and title bars all have an inactive shade (when window is not selected)
- - Compact variants need to shrink most padding and sizes.
+  - Shooting for 7.5:1 contrast ratios
```

Comment:
What is 7.5:1 contrast ratio? How would one confirm such a ratio?

Reply:
Addressed on line 104